R/scale-continuous.r

```diff
@@ -77,14 +77,14 @@ NULL
 scale_x_continuous <- function(name = waiver(), breaks = waiver(),
                                minor_breaks = waiver(), labels = waiver(),
                                limits = NULL, expand = waiver(), oob = censor,
-                               na.value = NA_real_, trans = "identity",
+                               na.value = NA_real_, trans = "identity", guide = waiver(),
```

Comment:
Somewhere in the code we should check and fail gracefully if the wrong type of guide is passed into the different scales. E.g. trying to use a `guide_legend()` with a `scale_x_continuous()` gives ` Error in f(..., self = self) : unused argument (breaks) `, instead of `Error: guide_legend() cannot be used with a position scale` (or something similar)